src/index.ts

```diff
@@ -2,3 +2,13 @@ import './polyfills'
 import './components/shop'
 import './components/checkout-button'
 import './shop'
+import { Shop } from './shop'
+
+// Shop.doSomething(Math.random())
+
+export { Shop } from './shop'
+console.log('################')
```

Comment:
Do you still need this log ^?

Reply:
yes, it's crucial :D 

Hehe, missed it during the clean-up. Buuuut you could also see it as a reminder that this is not working as it should. Will fix.